Cargo.toml

```diff
@@ -13,3 +13,4 @@ edition = "2021"
 crate-type = ["cdylib"]
 
 [dependencies]
+factorial = "0.2.1"
```

Comment:
It's not the kind of things we really pay attentions to so you we probably see this on every file file
For this kind of comment, left a comment on the PR rather than on a file. It will be more simple